docs/tutorials/user/credit-class-project-batch-management.md

```diff
@@ -0,0 +1,468 @@
+        "schema:version": ""
+      }
+    ],
+    "schema:url": ""
+  },
+  "regen:ecosystemType": [
+    ""
+  ],
+  "regen:verificationMethod": "",
+  "regen:projectActivities": [
+    ""
+  ]
+}
+```
+
+#### Projects
+
+The following includes a template for a project:
+
+```jsonld
+{
+  "@context": {
+    "schema": "http://schema.org/",
+    "regen": "https://schema.regen.network#",
+    "qudt": "http://qudt.org/schema/qudt/",
+    "unit": "http://qudt.org/vocab/unit/",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "regen:projectDesignDocument": {
+      "@type": "schema:URL"
+    },
+    "schema:url": {
+      "@type": "schema:URL"
+    },
+    "qudt:unit": {
+      "@type": "qudt:Unit"
+    },
+    "qudt:numericValue": {
+      "@type": "xsd:double"
+    },
+    "regen:projectStartDate": {
+      "@type": "xsd:date"
+    },
+    "regen:projectEndDate": {
+      "@type": "xsd:date"
+    },
+    "regen:offsetGenerationMethod": {
+      "@container": "@list"
+    }
+  },
+  "@id": "https://app.regen.network/project/<ID>",
+  "schema:name": "",
+  "schema:description": "",
+  "regen:projectDesignDocument": "",
+  "regen:projectDeveloper": {
+    "schema:name": "",
+    "schema:url": ""
+  },
+  "regen:projectType": "",
+  "regen:projectActivity": {
+    "schema:name": "",
+    "schema:url": ""
+  },
+  "regen:offsetGenerationMethod": [
+    ""
+  ],
+  "regen:projectSize": {
+    "qudt:unit": "",
+    "qudt:numericValue": 0
+  },
+  "regen:projectStartDate": "",
+  "regen:projectEndDate": "",
+  "schema:location": {
+    "@context": {
+      "type": "@type",
+      "@vocab": "https://purl.org/geojson/vocab#",
+      "coordinates": {
+        "@container": "@list"
+      },
+      "bbox": {
+        "@container": "@list"
+      }
+    },
+    "type": "Feature",
+    "place_name": "",
+    "geometry": {
+      "type": "Point",
+      "coordinates": [
+        0,
+        0
+      ]
+    }
+  },
+  "regen:offsetProtocol": {
+    "schema:name": "",
+    "schema:url": "",
+    "schema:version": ""
+  }
+}
+```
+
+#### Batches
+
+The following includes a template for a credit batch:
+
+```jsonld
+{
+  "@context": {
+    "schema": "http://schema.org/",
+    "regen": "https://schema.regen.network#",
+    "regen:additionalCertifications": {
+      "@container": "@list"
+    },
+    "regen:verificationReports": {
+      "@container": "@list"
+    },
+    "schema:url": {
+      "@type": "schema:URL"
+    }
+  },
+  "regen:verificationReports": [
+    {
+      "schema:url": ""
+    }
+  ],
+  "regen:projectVerifier": {
+    "schema:name": "",
+    "schema:url": {
+      "@type": "schema:URL",
+      "@value": ""
+    }
+  },
+  "regen:additionalCertifications": [
+    {
+      "schema:name": "",
+      "schema:url": {
+        "@type": "schema:URL",
+        "@value": ""
+      }
+    },
+    {
+      "schema:name": "",
+      "schema:url": {
+        "@type": "schema:URL",
+        "@value": ""
+      }
+    }
+  ]
+}
+```
+
+### IRI Generation
```

Comment:
yes that's the idea, but maybe using some REST endpoint would be more straightforward
wondering if we should turn the current iri-gen GET into a some new POST endpoint
having request body in HTTP GET is possible but seems quite controversial: https://stackoverflow.com/questions/978061/http-get-with-request-body
cc/ @wgwz

Reply:
I'm in favor of a REST endpoint and not requiring the user to install any packages in this tutorial.

Reply:
I also agree with using a `POST` endpoint as `GET` would require passing the JSON-LD as a parameter. We do the same in the data module for requests that would otherwise require url encoding.

Reply:
Not sure what to do about this in the meantime until we have the new endpoint available.

Reply:
Even though people say it's controversial, it's quite reasonable in this IMO, because it's just effectively an RPC call, that is doing a stateless data transformation. IMO it's "really bad" only when your doing something stateful with that data from the request body. Because GET's are stateless. I think it's fine, it's effectively like a request replay service (https://reqbin.com/). IMHO it works quite nicely for this use-case, and no need to worry.

Reply:
that said, we can trivially switch to a new `POST` endpoint but IMO it's a nit. i'm sure there's plenty of people that would disagree with me though.

Reply:
Regardless of whether or not we transition the current `GET /iri-gen` endpoint to a `POST /iri-gen`, I think for the purposes of this PR we can move forward with suggesting users use the `GET /iri-gen` endpoint with a request body.  As @blushi and @wgwz point out, it's controversial, but I think it is technically supported and should work as expected. 

We can then move the conversation of changing our REST vocabulary (e.g. switching the current GET to a POST), to a new issue on the regen registry server, and just remember to update these docs if/when we change the API functionality on the registry-server.

Reply:
Ok, I've updated `POST` to `GET` for the IRI generation endpoint. https://github.com/regen-network/regen-ledger/pull/1965/commits/a4a221eb7873d041cbe6d95887944f28b1ffbaf3